src/Controls/src/Core/Button.cs

```diff
@@ -38,7 +38,7 @@ public partial class Button : View, IFontElement, ITextElement, IBorderElement,
 
 		public static readonly BindableProperty FontAutoScalingEnabledProperty = FontElement.FontAutoScalingEnabledProperty;
 
-		public static readonly BindableProperty BorderWidthProperty = BindableProperty.Create("BorderWidth", typeof(double), typeof(Button), -1d);
+		public static readonly new BindableProperty BorderWidthProperty = BindableProperty.Create("BorderWidth", typeof(double), typeof(Button), -1d);
```

Comment:
Can we just delete all of these? What's the reason to keep any of these around now that the base class has this same property with the same type